lib/modules/datasource/docker/index.ts

```diff
@@ -714,7 +714,16 @@ export class DockerDatasource extends Datasource {
         headers,
         noAuth: true,
       });
-      labels = JSON.parse(configResponse.body).config.Labels;
+
+      const body = JSON.parse(configResponse.body);
+      if (body.config) {
+        labels = body.config.Labels;
+      } else {
+        logger.debug(
+          { configResponse },
```

Comment:
```suggestion
          { config: body },
```

only log the body, not the full request